frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/base/built_in_page_widget.dart

```diff
@@ -54,10 +54,14 @@ class _BuiltInPageWidgetState extends State<BuiltInPageWidget> {
           if (board != null) {
             return _build(context, board);
           }
+          return const SizedBox.shrink();
```

Comment:
I'm not sure I'm a fan of this solution.

I would prefer that deleting a Page would also remove it from any documents which might reference it.

Reply:
IIRC, the backend can either return a board, null, or error so the widget still needs to account for this case. This solution will do for now.